core/src/main/java/nl/nn/adapterframework/configuration/filters/SkipContainersFilter.java

```diff
@@ -27,6 +27,10 @@
 
 public class SkipContainersFilter extends FullXmlFilter {
 
+	// The Frank!Doc trusts that this class is used to remove <Module> element.
+	// If you ever want to remove Module from this list, please update the Frank!Doc.
+	// To search for the relevant Frank!Doc code, you can start at
+	// nl.nn.adapterframework.frankdoc.Constants.MODULE_ELEMENT_NAME.
```

Comment:
Als je commentaar bij deze class zet, dan moet dat eerst beschrijven wat deze class doet. Daarna kan je dan een opmerking zoals deze maken. Nu lijkt het er op dat dit het belangrijkste is.

Reply:
Gedaan. Er is nu ook commentaar dat beschrijft wat de class doet.